juju-webapp/src/main/java/com/ubiqube/juju/service/WorkspaceService.java

```diff
@@ -304,6 +306,74 @@ public ProcessResult status() {
 		return run(builder);
 	}
 
+	public boolean isK8sReady() {
+		ProcessResult result = status();
+		LOG.info("isK8sReady result", result.getStdout());
+		try {
+			JSONObject jsondata = new JSONObject(result.getStdout());
+			JSONObject applications = jsondata.getJSONObject("applications");
+			for (String appName : applications.keySet()) {
+				JSONObject application = applications.getJSONObject(appName);
+				JSONObject appStatus = application.getJSONObject("application-status");
```

Comment:
With jackson you can use basic java types `Map`, `List`, ... But there is also some special nodes. But the first mode doesn't require any dependencies.